crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -119,6 +129,10 @@ def validate_proposal_after_save proposal
 
     validate_minimum_three_nodes_in_cluster(proposal)
 
+    unless proposal["deployment"]["ceilometer"]["elements"]["ceilometer-hyperv-agent"].empty? || hyperv_available?
```

Comment:
undefined method `empty?' for nil:NilClass

Throws an error in the crowbar when applying the ceilometer-hyperv-agent.
![latest-screenshot](https://cloud.githubusercontent.com/assets/476002/7607017/67f3defa-f95e-11e4-833a-ffc103536eaa.png)